utils/vcsdetails.go

```diff
@@ -89,12 +89,12 @@ func (this *VcsCache) clearCacheIfExceedsMax() {
 	}
 }
 
-func tryGetGitDetails(path string, this *VcsCache) (string, string, error) {
-	exists, err := fileutils.IsDirExists(filepath.Join(path, ".git"), false)
+func tryGetGitDetails(path string) (string, string, error) {
```

Comment:
Looks like this fucntion no longer needs to return an error.

Reply:
`extractGitDetails` may still return an error.